PubHub/PubHub.Common/Services/IHttpClientService.cs

```diff
@@ -15,15 +15,15 @@ public interface IHttpClientService
         /// <param name="uri">URI (must NOT start with '/').</param>
         /// <param name="httpContent">Content of the POST message.</param>
```

Comment:
We should maybe update the `param` name to say `content` instead of the old name 🤔

Reply:
Noted. This should be changed now